internal/slicer/slicer_test.go

```diff
@@ -805,6 +863,12 @@ func (a *testArchive) Exists(pkg string) bool {
 	return ok
 }
 
+func (a *testArchive) Info(pkg string) (*archive.PackageInfo, error) {
+	// TODO Currently we do not need PackageInfo for slicer_test. Hence, we are
+	// returning nil here. In future, add necessary structs as needed.
```

Comment:
What about panicking then to make it even more explicit if somebody is writing new tests? If we end up extracting the testarchive then we might get the Info for "free" instead.

Reply:
This is not applicable anymore due to refactoring to `testutil.TestArchive`.